crates/iceberg/src/lib.rs

```diff
@@ -18,6 +18,7 @@
 //! Native Rust implementation of Apache Iceberg
 
 #![deny(missing_docs)]
+#![feature(test)]
```

Comment:
Hi, iceberg needs to run on stable Rust, so we can't introduce nightly features. Please consider moving those benchmarks out.